test/problematic_tests/impl_test.py

```diff
@@ -0,0 +1,48 @@
+import os
+import pytest
+import unittest
+
+from GenomeFileUtil.GenomeFileUtilImpl import GenomeFileUtil
+from conftest import assert_exception_correct
+
+
+class ImplTest(unittest.TestCase):
+    def setUp(self):
+        self.cfg = {}
+        self.cfg["workspace-url"] = "https://ci.kbase.us/services/ws"
+        self.cfg["shock-url"] = "https://ci.kbase.us/services/shock-api"
+        self.cfg["handle-service-url"] = "https://ci.kbase.us/services/handle_service"
+        self.cfg["SDK_CALLBACK_URL"] = "http://ip:port"
+        self.cfg["scratch"] = "/kb/module/work/tmp"
+        self.cfg["srv-wiz-url"] = "https://ci.kbase.us/services/service_wizard"
+        self.cfg["auth-service-url"] = "https://ci.kbase.us/services/auth/api/legacy/KBase/Sessions/Login"
+        self.cfg["re-api-url"] = "https://ci.kbase.us/services/relation_engine_api"
+        self.cfg["KB_AUTH_TOKEN"] = os.environ.get("KB_AUTH_TOKEN", None)
+
+    def tearDown(self):
+        for kbase_secure_param in (
+            "KBASE_SECURE_CONFIG_PARAM_MAX_THREADS",
+            "KBASE_SECURE_CONFIG_PARAM_THREADS_PER_CPU",
+        ):
+            if kbase_secure_param in os.environ:
+                del os.environ[kbase_secure_param]
```

Comment:
```suggestion
            os.environ.pop(kbase_secure_param, None)
```
No real need for the `if`, the end result is the same

Reply:
updated!

Reply:
:+1: